Makefile

```diff
@@ -1,13 +1,17 @@
 .PHONY: help clean purge compile-contracts patch-ipfs mk-build-dir copy-misc copy-backend compile-js copy-frontend archive
 
-export NODE_ENV ?= development
+export NODE_ENV ?= localhost
 export WALLET_PASSWORD ?= dev_password
 export WALLET_MNEMONIC ?= erupt point century seek certain escape solution flee elegant hard please pen
 
 ifeq ($(NODE_ENV),production)
 export EMBARK_TARGET ?= livenet
 else
-export EMBARK_TARGET ?= testnet
+  ifeq ($(NODE_ENV), localhost) 
```

Comment:
`ifeq` and other `Makefile` statements like that should not be indented, this isn't valid syntax.